client/state/account-recovery/actions.js

```diff
@@ -7,6 +7,22 @@ import {
 	ACCOUNT_RECOVERY_FETCH,
 	ACCOUNT_RECOVERY_FETCH_SUCCESS,
 	ACCOUNT_RECOVERY_FETCH_FAILED,
+
+	ACCOUNT_RECOVERY_PHONE_UPDATE,
+	ACCOUNT_RECOVERY_PHONE_UPDATE_SUCCESS,
+	ACCOUNT_RECOVERY_PHONE_UPDATE_FAILED,
+
+	ACCOUNT_RECOVERY_PHONE_DELETE,
+	ACCOUNT_RECOVERY_PHONE_DELETE_SUCCESS,
+	ACCOUNT_RECOVERY_PHONE_DELETE_FAILED,
+
+	ACCOUNT_RECOVERY_EMAIL_UPDATE,
+	ACCOUNT_RECOVERY_EMAIL_UPDATE_SUCCESS,
+	ACCOUNT_RECOVERY_EMAIL_UPDATE_FAILED,
+
+	ACCOUNT_RECOVERY_EMAIL_DELETE,
+	ACCOUNT_RECOVERY_EMAIL_DELETE_SUCCESS,
+	ACCOUNT_RECOVERY_EMAIL_DELETE_FAILED,
```

Comment:
This seems like a lot of action types, do they all need to be separate types? Is there a way these could be more generalized, using action parameters instead? For example
`ACCOUNT_RECOVERY_UPDATE`, then passing `{ email: ... }` or `{ phone: ... }`